src/tools/_tools.utils.scss

```diff
@@ -1,3 +1,4 @@
+@use "sass:math";
```

Comment:
To keep the scope if this PR low, use `@use` where necessary and replace `/` division with `math.div` seems fine.

That said I think it is worth to evaluate the Sass module system and how it can be utilized within `fluidms` and a follow-up.

Reply:
First line for `@use` is ok? Or should it be included after comment block which documents the code below?